_data/projects/linkin.yml

```diff
@@ -0,0 +1,16 @@
+name: Linkin
+desc: Linkin is a customizable self hosted link tree platform
+site: http://linkindemo.vercel.app/
+tags:
+  - reactjs
+  - nextjs
+  - node.js
+  - platform
+  - postgres
+  - hacktoberfest
+  - linktree
+  - linktree-alternative
+  - vercel
+upforgrabs:
+  name: good first issue
+  link: https://github.com/RizkyRajitha/linkin/labels/good%20first%20issue
```

Comment:
:art: at the moment there are no issues found with this label, which means this won't be prioritized when this is merged. Were you planning to add any tasks to this label?